### PR TITLE
Show number of hours indicated in availability

### LINKED ISSUE #204: Show number of hours indicated in availability
Show the number of free hours that each supervisor has indicated in their Schedule, preferably in sorted order.

This would help Ops in scheduling as supervisors that have less number of available hours can be scheduled first, since they have less flexibility to be moved around later.

Not sure how this info should be presented or what the interface should look like :/

### PR DESCRIPTION
**Pull request by chrisjwelly**
Fixes #204

### CODE REVIEW

File: app/helpers/availabilities_helper.rb [hunk 1]
```diff
@@ -74,4 +74,16 @@ def choose_form_path
       availabilities_path
     end
   end
+
+  def total_user_hours
+    relevant_duties = Availability.includes(:time_range)
```
Comment (indocomsoft) [inline app/helpers/availabilities_helper.rb]: Should instead be `current_user.availabilities.includes(:time_range)`

File: app/helpers/availabilities_helper.rb [hunk 2]
```diff
@@ -15,9 +15,9 @@ def generate(availability)
 
   def stylise_user(user)
     if user[:mc]
-      content_tag(:span, user[:username], class: 'mc')
+      content_tag(:span, user[:username] + "(10)", class: 'mc')
```
[the remark this answers is not in the document]
Comment (chrisjwelly) [reply app/helpers/availabilities_helper.rb]: Sorry, forgot to remove this after testing if we could display the number of hours through this

File: app/helpers/availabilities_helper.rb [hunk 3]
```diff
@@ -74,4 +74,16 @@ def choose_form_path
       availabilities_path
     end
   end
+
+  def total_user_hours
+    relevant_duties = Availability.includes(:time_range)
+                                  .where(user_id: current_user.id,
+                                 date: @start_date..@end_date,
```
[the remark this answers is not in the document]
Comment (chrisjwelly) [reply app/helpers/availabilities_helper.rb]: Should be `user.availabilities.includes(:time_range`

File: app/views/availabilities/show_everyone.html.erb [hunk 4]
```diff
@@ -1,5 +1,10 @@
 <div class="container-fluid">
   <h1>Schedule Availability for all users</h1>
+  <div>
+    <body> <strong>Format</strong>: Name (Number of available hours indicated)
```
Comment (indocomsoft) [inline app/views/availabilities/show_everyone.html.erb]: `<body>` inside HTML is not valid HTML (although valid HTML is arguable but...)

**Pull request merged**